songs/songs.spec.js

```diff
@@ -0,0 +1,29 @@
+const request = require("supertest");
+const server = require("../api/server");
+const Songs = require("../data/db_config");
+const model = require("./songs_model");
+// var knexCleaner = require("knex-cleaner");
+
```

Comment:
Glad to see coverage % climbing!! 🚀
Let's change the name of the router files and their test counterparts to match one convention.
songs/songs_router
songs/songs_router.spec.js
&
videos/videos_router
videos/videos_router.spec.js
are the only two offending pairs I'm seeing.

Reply:
These changes are implemented in a current PR. Should come in after this is merged.